### PR TITLE
DE-752 | `multi_delimiter` analyzer [3.12]

### PR DESCRIPTION
**Pull request by aMahanna**
Can be merged after `3.12` image is introduced

tested to work locally with `3.12.0-NIGHTLY.20240305`

### CODE REVIEW

File: tests/test_analyzer.py [hunk 1]
```diff
@@ -71,3 +81,16 @@ def test_analyzer_management(db, bad_db, cluster, enterprise, db_version):
             "format": "latLngDouble",
         }
         assert db.delete_analyzer(analyzer_name)
+
+    # Test create delimieter analyzer with multiple delimiters
+    # NOTE: This is currently failing on 3.12.0-NIGHTLY.20240116
```
Comment (aMahanna) [inline tests/test_analyzer.py:86]: highlighting

File: tests/test_analyzer.py [hunk 2]
```diff
@@ -71,3 +81,16 @@ def test_analyzer_management(db, bad_db, cluster, enterprise, db_version):
...
+    if db_version >= version.parse("3.12.0"):
+        result = db.create_analyzer(
+            name=generate_analyzer_name(),
+            analyzer_type="delimiter",
```
Comment (alexbakharew) [inline tests/test_analyzer.py:90]: The analyzer type here is wrong. It should be `multi_delimiter`.
Here is the reference: https://docs.arangodb.com/3.12/index-and-search/analyzers/#multi_delimiter

Comment (aMahanna) [reply tests/test_analyzer.py]: thanks fixed in b48e105

File: tests/test_analyzer.py (hunk 1 above)
Comment (aMahanna) [reply tests/test_analyzer.py]: resolved after testing with `3.12.0-NIGHTLY.20240305`

**Review by apetenchea — APPROVED** [review]
LGTM

**Pull request merged**